src/tss2-fapi/fapi_crypto.c

```diff
@@ -876,15 +876,14 @@ ecdsa_verify_signature(
                      "Invalid ECDSA DER signature", error_cleanup);
     }
 
-    eccKey = EVP_PKEY_get1_EC_KEY(publicKey);
+    eccKey = EVP_PKEY_get0_EC_KEY(publicKey);
```

Comment:
Looks like there is no EVP_PKEY_get0_EC_KEY or EVP_PKEY_get0_RSA in OpenSSL 1.0.2 that comes with ubuntu 16

Reply:
Oh yeah, easy enough to add compat layer. I'm surprised they have they 1 versions in 1.0.2.

Reply:
Probably better just to leave that in place, and when OSSL < 1.1.0 support is no longer needed, we can just update it. Id rather have consistent code, than adding a compat layer. The other patch(s) make sense though